wiki/Beatmapping/Beatmap_submission/de.md

```diff
@@ -0,0 +1,27 @@
+# Einreichen
+
+[Beatmaps](/wiki/Beatmap) können auf die osu!-Webseite über den [Editor im Spiel](/wiki/Client/Beatmap_editor) eingereicht werden. Durch das Einreichen von Beatmaps können andere Nutzer ein Augenmerk auf diese werfen und die Beatmaps möglicherweise in die Kategorien [Ranked](/wiki/Beatmap/Category#ranked) oder [Loved](/wiki/Beatmap/Category#loved) eintreten.
```

Comment:
```suggestion
[Beatmaps](/wiki/Beatmap) können auf die osu!-Webseite über den [Editor im Spiel](/wiki/Client/Beatmap_editor) eingereicht werden. Durch das Einreichen von Beatmaps können andere Nutzer einen Blick auf diese werfen und die Beatmaps möglicherweise in die Kategorien [Ranked](/wiki/Beatmap/Category#ranked) oder [Loved](/wiki/Beatmap/Category#loved) eintreten.
```
I don't believe `Ein Augenmerk auf etwas werfen` is a real idiom. The only one I was able to find was `sein Augenmerk auf etwas richten`, which would be focusing on a particular detail and therefore not really applicable here.